README.min.md

```diff
@@ -0,0 +1,32 @@
+# Local Deployment of Minimal SQuARE
+
+This branch hosts a minimal version of SQuARE for the convenience of local deployment.
+
+To get started, please run:
+```
+pip install -r local_deploy/requirements.txt
+bash install-local.sh
+```
+to generate a `docker-compose.yaml` file (and a authentication token for local deployment in `./local_deploy`)
+
+And then:
+```bash
+docker-compose up -d
+cd local_deploy
+python deploy_ds.py  # Tested with Python 3.7.13 
+```
+
+This will start the minimal SQUARE service and run the example script `local_deploy/deploy_ds.py` to 
+- Download example corpus scifact;
+- Upload the corpus (to the ES in SQUARE);
+- Do BM25 search;
+- Embed and index the corpus with DPR (GPU 1min, CPU 30min);
+- Start a Faiss container to load the built dense index;
+- Do dense search.
+
+One can also get access to the [SWAGGER UI](https://swagger.io/tools/swagger-ui/) page for the interactive API. To do so, one can do 
+```bash
+cd local_deploy
+python get_token.py
```

Comment:
Could you please provide more details? Cannot really understand this comment

Reply:
once the venv with square-auth installed is activated, you can use `square_token` to generate a token. The get_token.py file is therefore not requried.

Reply:
Cool. Now it uses `square_auth.utils.generate_token` to replace the re-implementation of this method. I would suggest keeping such a script file, since people might not know this python entry point and this can also work as an example. [8f494be](https://github.com/UKP-SQuARE/square-core/pull/354/commits/8f494be447b2c3e02ee4b3be043bfc5a898e63c6)

Reply:
I have also added a hint in the readme about this `square_token` entry point [4aa24cc](https://github.com/UKP-SQuARE/square-core/pull/354/commits/4aa24ccaac468fcbfbf6540cb451282ee34cb844)